### PR TITLE
Update initial blog post with 0.5.0 release info

### PR DESCRIPTION
**Pull request by pkosiec**
## Description

Changes proposed in this pull request:

- Reduce blog post introduction by 18.5% (from 269 words to 219 words)
- Update blog post with release info
- Update website banner

## Preview

https://6ddcf47b.capact-website.pages.dev/blog

## Related issue(s)

https://github.com/capactio/capact/issues/486

### CODE REVIEW

File: blog/2021-09-07-introducing-capact.md [hunk 1]
```diff
@@ -12,16 +12,15 @@ tags:
...
+Sharing expertise is something that already happens—we have plenty of libraries and frameworks out there. But what if we went a step further and have a way to create, use and share building blocks that are **language-agnostic abstracted capabilities**?
 
-For example, if you're not a cloud expert, all you need to know is that you want a managed PostgreSQL database on AWS and let others make it. In that way, services are managed by subject-matter experts, and you can simply focus on your business logic. Likewise, if you need any Kubernetes cluster—by saying "I want any Kubernetes cluster" and letting the magic happen, you could cover both local development and production scenarios...
+For example, if you're not a cloud expert, all you need to know is that you want a managed PostgreSQL database on AWS. Let the subject-matter experts take care of it, and simply focus on your business logic. Likewise, if you need any Kubernetes cluster—by saying "I want any Kubernetes cluster" and letting the magic happen, you could cover both local development and production scenarios…
 
-What if I told you... **it is all possible**?
+What if I told you… **it is all possible**?
```
Comment (platten) [inline blog/2021-09-07-introducing-capact.md:23]: it is all possible --> Capact makes it possible.

**Review by platten — CHANGES_REQUESTED** [review]
One small change and LGTM after that.

File: blog/2021-09-07-introducing-capact.md (hunk 1 above)
Comment (pkosiec) [reply blog/2021-09-07-introducing-capact.md:23]: There's the following sentence right after "What if I told you… it is all possible?":

> Introducing Capact, crowdsourced expertise. Learn once, use everywhere.

🤔 Personally I like the current version more, as at this point the reader doesn't know what's Capact.

Maybe we could keep it as it is, or alternatively have something like:

"What if I told you… there's a project that makes it possible?"

Comment (platten) [reply blog/2021-09-07-introducing-capact.md:23]: Good point. Let's go with: "now this is all possible."

Comment (pkosiec) [reply blog/2021-09-07-introducing-capact.md:23]:
```suggestion
What if I told you… **now this is all possible**?
```

**Comment by pkosiec** [comment]
I will merge the PR once we have Capact 0.5.0 successfuly released 🙂

**Pull request merged**